src/core/components/accordion/accordionItem/accordionItem.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
+import { forwardRef, type ReactNode } from 'react';
+import { AccordionContainer, type IAccordionContainerProps } from '../accordionContainer/accordionContainer';
+import { AccordionItem } from './accordionItem';
+
+describe('<AccordionItem /> component', () => {
+    const MockChildren = forwardRef<HTMLButtonElement, { children?: ReactNode }>((props, ref) => (
+        <button ref={ref}>{props.children}</button>
+    ));
+    MockChildren.displayName = 'MockHeader';
+
+    const createTestComponent = (values?: Partial<IAccordionContainerProps>) => {
```

Comment:
The `createTestComponent` function should always get the properties of the component we're testing. In that way, if we add one property that we need to test, we already have everything setup to test it.